inst/extdata/energy/A32.nonenergy_Cseq.csv

```diff
@@ -0,0 +1,8 @@
+# File: A32.nonenergy_Cseq.csv,,,,
```

Comment:
Here and in other files, please remove ugly Excel trailing commas in header lines